networking_cisco/plugins/cisco/cfg_agent/device_drivers/asr1k/aci_asr1k_routing_driver.py

```diff
@@ -463,9 +536,9 @@ def _set_vrf(self, ri, port, config):
 
     def _set_vrf_pid(self, ri, port, config):
         # Convert a VRF to a process iD for a router instance
-        vrf = self._get_vrf_name(ri)
+        vrf = self._get_vrf_name(ri).strip('nrouter-')[:6]
```

Comment:
do you have to limit it to the first 6 characters here? It seems to me that u can keep the whole string here coz later on u do the stripping in line#543 anyway.